### PR TITLE
config file added and applied to pouch service

### PR DESCRIPTION
**Pull request by niczem**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -30,6 +30,7 @@
       </div>
     </app-root>
 
+    <script src="config/config.js"></script>
```
Comment (dubbl) [inline src/index.html]: does this file exist? is it generated?

Comment (niczem) [inline src/index.html]: good point... added :)

File: src/app/services/config.service.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+@Injectable()
+export class ConfigService {
+constructor(private http:Http) {}
+
+private result: Object;
+
+getConfiguration(key) { 
+    return app_config[key]
+}
+}
```
Comment (koop-bremen) [inline src/app/services/config.service.ts]: Maybe I'm a bit picky here, but according to the POSIX standard  lines should always end with a newline character, so I would suggest we stick to that. If you work with command line tools this is really helpful.

File: src/config/config.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+var app_config = {
+  "db_remote_url":"http://localhost:5984/"
+}
```
Comment (koop-bremen) [inline src/config/config.js]: Same applies here.

Comment (niczem) [inline src/config/config.js]: done :)

File: src/app/services/auth.service.ts [hunk 4]
```diff
@@ -18,13 +19,18 @@ export class AuthService {
...
+  constructor(PouchService:PouchService, ConfigService:ConfigService) {
 
     console.log('initializing login...');
 
     //this.db = PouchService.initDB('mydb', {skipSetup:true});
 
-    this.db = new PouchDB('http://localhost:5984/mydb', {skipSetup: true});
+    this.remote = ConfigService.getConfiguration('db_remote_url');
+
+    this.db = new PouchDB(this.remote+'_users/', {skipSetup: true});
```
Comment (bernd) [inline src/app/services/auth.service.ts]: This requires that `this.remote` always has a trailing `/` and it breaks if not. Maybe there is a way to handle that automatically.

Comment (niczem) [inline src/app/services/auth.service.ts]: fixed:
`this.remote = this.remote.replace(/\/?$/, '/');`

**Pull request merged**